### PR TITLE
fix(prescaling): update a few things

### PR DESCRIPTION
**Pull request by taufort**
# Why?

* rename the wall image used for the prescaling blog post to attribute the image to unsplash and to the real author
* add a note with a link to another blog post on Kubernetes
* fix a typo 'finale' => 'final'

### CODE REVIEW

File: _posts/2022-02-03-prescaling.md [hunk 1]
```diff
@@ -55,8 +55,12 @@ adjusts the size of a Kubernetes cluster by adding/removing nodes.
 pods (and new “useless” nodes, as a consequence), so the cluster has available capacity that will be used to start 
 applications pods quicker.
 
+> If you wish to know more about which tools we use and why we use them in our Kubernetes clusters, I advise you to 
+check out another dedicated blog post named 
+["Three years running Kubernetes on production at Bedrock"](https://tech.bedrockstreaming.com/2020/12/08/Three-years-running-kubernetes-on-production-at-Bedrock.html).
+
 Unfortunately, all those tools are not sufficient to deal with heavy and sudden traffic spikes on some special 
-evenings such as the finale of a successful show. During this kind of events, users arrive massively, all at the 
+evenings such as the final of a successful show. During this kind of events, users arrive massively, all at the
```
Comment (outadoc) [inline _posts/2022-02-03-prescaling.md:63]: I think `finale` is the right term in this case

Comment (taufort) [reply _posts/2022-02-03-prescaling.md:63]: I double checked.  `finale` is more appropriate for the end of a musical or a show. Here, I'm talking about the final of Top Chef or the final of the UEFA.

Comment (taufort) [reply _posts/2022-02-03-prescaling.md:63]: Maybe I can precise that it could be the `final of a football game`? :shrug:

Comment (outadoc) [reply _posts/2022-02-03-prescaling.md:63]: Sounds a bit better to me. Didn't mean to interrupt your flow by the way, sorry 🙂

**Review by Oliboy50 — APPROVED** [review]

**Review by vgallissot — APPROVED** [review]

File: _posts/2022-02-03-prescaling.md (hunk 1 above)
Comment (taufort) [reply _posts/2022-02-03-prescaling.md]: Fixed!

Oh no worries, that's what reviews are for :wink:

**Pull request merged**